packages/container/lib/container.js

```diff
@@ -191,15 +280,47 @@ function lookup(container, fullName, options = {}) {
     return container.cache[fullName];
   }
 
-  let value = instantiate(container, fullName);
+  return instantiateFactory(container, fullName, options);
```

Comment:
yes, we should only have 1 behavior in the system at any point in time, once we cut over it should be all no double extend.